teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
```

Comment:
- [ ]  The parent_permission parameter should be defined correctly in the super method to avoid errors. It uses a colon for assignment so the way to assign it would be different from other parameters.

`super(age, name, parent_permission: parent_permission)`